pkg/testcase/clusterreset.go

```diff
@@ -0,0 +1,98 @@
+package testcase
+
+import (
+	"fmt"
+
+	"github.com/rancher/distros-test-framework/factory"
+	"github.com/rancher/distros-test-framework/shared"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+func TestClusterReset() {
+	cluster := factory.ClusterConfig(GinkgoT())
+
+	killall(cluster)
+	shared.LogLevel("INFO", "%s-service killed", cluster.Config.Product)
+
+	stopServer(cluster)
+	shared.LogLevel("INFO", "%s-service stopped", cluster.Config.Product)
+
+	productLocationCmd := fmt.Sprintf("sudo find / -type f -executable -name %s", cluster.Config.Product)
```

Comment:
Suggestion: Some of these vars can be re-used and need no new assignment
- `productLocationCmd` can be just `cmd`
- `productLocation, productLocationErr` can be `res,err`
- re-use `cmd` instead of `resetCmd`
- re-use `cmd,res` instead of `resetRes, resetCmdErr`

Reply:
Similar suggestion can be applied on other func as necessary

Reply:
for stylistic reasons I want to keep it this way to specify what the ```cmd``` is related to and doing

Reply:
Ok but the command itself is explaining what it is doing and it is not necessary to define it again... just my thoughts as I think we should prioritize resource allocation and reusability over styling

Reply:
I am still reluctant to change it as I also received comments to label the commands like ```killallcmd``` ```resetcmd``` ass they have been rather than having ```cmd``` throughout the entire clusterreset.go file